.ember-cli

```diff
@@ -5,5 +5,6 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false
+  "disableAnalytics": false,
+  "port": 8080
```

Comment:
yes, sorry, that's left over from my local.  this isn't 100% ready to go in anyway, I just wanted to post to get some feed back.